cfme/automate/service_dialogs.py

```diff
@@ -136,6 +138,18 @@ def element_type(self, each_element):
                                 'entry_description': "entry_desc"})
         if choose_type == "Text Area Box":
             text_area_table.click_cell(1, value="Default text")
+        if choose_type == "Drop Down Dynamic List":
+            node1 = each_element.get("field_entry_point")
+            sel.click(element_form.field_entry_point)
+            tree = Tree({
```

Comment:
Please define this Tree up above with the other locators, rather than inside this function.